io.github.rsvzz.csven.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "io.github.rsvzz.csven",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "csven",
+  "finish-args": [
+      "--share=ipc",
+      "--share=xdg-desktop-portal-gnome",
```

Comment:
I don't know what documentation told you to add this, but it's invalid and the build will definitely fail.

Also you removed the permissions to display.